alpine/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM alpine:3.6
 LABEL maintainer Marco Palladino, marco@mashape.com
```

Comment:
are we keeping this as @mashape.com? (same below)

Reply:
Ideally not - also not an individual, this should be a team or support email. Pinged @salazar about it before, I think this got queued up

Reply:
My personal opinion is that we should either use our team alias or, if it's not supposed to be made public, a team-specific support address.